letsdrill.rb

```diff
@@ -1,24 +1,30 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  letter = 'F'
+  if integer >= 90
+    letter = 'A'
+  elsif integer >= 80
+    letter = 'B'
+  elsif integer >= 70
+    letter = 'C'
+  elsif integer >= 60
+    letter = 'D'
+  end
+  letter
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+  array.min_by{ |e| e.length}
```

Comment:
Doesn't get much simpler than that!

After folks get used to writing Ruby for a while, they tend to write this like: `array.min_by(&:length)`. You don't really need to rewrite it, I just wanted to let you know about the other style. The punctuation is a bit of a distraction, but I think it is better because you don't need to think about any particular instance, you're just saying "use the length method".

If want to get into a bit of rabbit hole, you could look into how that syntax works. (It's called `Symbol#to_proc`.)